alpha/scripts/utils/mergeNewlyCreatedDuplicatedUsers.php

```diff
@@ -7,25 +7,37 @@
 define('K2_KUSER', 'k2');
 define ('MAX_USERS_TO_HANDLE', 10000);
 
-$fp = fopen(__DIR__ . '/mergeNewlyCreatedDuplicatedUsers.php', "r+");
-if (!flock($fp, LOCK_EX|LOCK_NB))
+try
 {
-	KalturaLog::debug('Could not lock file. Exiting.');
-	return;
-}
-$lastRunFilePath = $argv[1];
+	$fp = fopen(__DIR__ . '/mergeNewlyCreatedDuplicatedUsers.php', "r+");
+	if (!flock($fp, LOCK_EX|LOCK_NB))
+	{
+		KalturaLog::debug('Could not lock file. Exiting.');
```

Comment:
What if we cannot run the script - how can we tell about it ? throw exception here.